src/main.py

```diff
@@ -2,15 +2,26 @@
 main file
 """
 from school import Classroom, Plotter, import_results
-from school.teachers import RandomTeacher, BaseTeacher
+from school.teachers import *
 from school.students import RashStudent
 
+import numpy as np
+
 if __name__ == '__main__':
-    c = Classroom(7, BaseTeacher, RashStudent, nStudents=100, estimateDifficulty=False)
-    c.run(timeToExam=100, numberOfIteration=10, saveResults=True, visualiseResults=False, savePlot=True)
+    timeToExam = 10
+    c = Classroom(1, DQNTeacher, RashStudent, timeToExam=timeToExam, nStudents=100, gamma=0.99, nLast=5, learning_rate=0.05,
+                  verbose=False, start_of_random_iteration=10, number_of_random_iteration=5, cnn=True)
```

Comment:
Niepotrzebnie zostawiałem parametry od AC, jak DQN jest wywoływany, ale mówi się trudno

Reply:
Czy tam RandomTeacher, bo znowu coś testowałem i zapomniałem zmienić xD

Reply:
To nie ma większego znaczenia i tak każdy jak to testuje to sobie zmienia.